projects/tuist/fixtures/ios_app_with_headers_in_one_dir/App/Config/App-Info.plist

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need those PLISTs? Can't tuist generate them?

Reply:
I just copied another fixture